app/src/main/res/layout/fragment_preview_slider.xml

```diff
@@ -24,82 +24,22 @@
     android:background="@color/previewBackground"
     tools:context="com.infomaniak.drive.ui.fileList.preview.PreviewSliderFragment">
 
-    <androidx.viewpager2.widget.ViewPager2
-        android:id="@+id/viewPager"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent" />
-
     <androidx.constraintlayout.widget.ConstraintLayout
-        android:id="@+id/header"
+        android:id="@+id/pdfContainer"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content">
-
-        <com.google.android.material.button.MaterialButton
-            android:id="@+id/backButton"
-            style="@style/IconButton"
-            android:layout_width="50dp"
-            android:layout_height="50dp"
-            android:layout_marginStart="@dimen/marginStandardMedium"
-            android:layout_marginTop="@dimen/marginStandardMedium"
-            android:contentDescription="@string/buttonBack"
-            app:backgroundTint="@color/previewBackgroundTransparent"
-            app:icon="@drawable/ic_chevron_left"
-            app:iconSize="20dp"
-            app:iconTint="@android:color/white"
-            app:layout_constraintEnd_toStartOf="@+id/pageNumberChip"
-            app:layout_constraintHorizontal_bias="0.0"
-            app:layout_constraintHorizontal_chainStyle="spread_inside"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
+        android:layout_height="match_parent">
 
```

Comment:
You don't need a ConstraintLayout for this simple use, a FrameLayout should be enough